webapp/apps/taxbrain/helpers.py

```diff
@@ -162,8 +165,8 @@ def string_to_float_array(s):
     'df_dec': 'Difference between Base and User plans by expanded income decile',
     'mX_bin': 'Base plan tax vars, weighted avg per expanded income bin',
     'mY_bin': 'User plan tax vars, weighted avg per expanded income bin',
-    'df_bin': 'Difference between Base and User plans by expanded income bin',
-    'fiscal_tots': 'Total Revenue Change by Calendar Year',
+    'df_bin': 'Individual Income Tax: Difference between Base and User plans by AGI decile',
```

Comment:
Yes, should be expanded income bin for now. That said, I hope we can generalize this later on so the user can decide whether to use _expanded_income or c00100 as the tab variable for tables. https://github.com/OpenSourcePolicyCenter/webapp-public/issues/35#issuecomment-148811619